api/token.go

```diff
@@ -558,6 +551,7 @@ func (a *API) issueRefreshToken(ctx context.Context, conn *storage.Connection, u
 		TokenType:    "bearer",
 		ExpiresIn:    config.JWT.Exp,
 		RefreshToken: refreshToken.Token,
+		User:         user,
```

Comment:
basically add the user object to the token struct returned so calling `issueRefreshToken` will now embed the user object in the struct as well